### DISCUSSION TITLE
Why does my LSPosed show that it is not installed?

### DISCUSSION THREAD

**Discussion by Sunrongguo2008** [LSPosed/LSPosed#2214] [Q&A] [answered]
![屏幕截图(2)](https://user-images.githubusercontent.com/91547012/200537732-f4ef3abd-260e-485c-ab07-610f4a79ddfa.png)

**Comment by Sunrongguo2008** [comment]
Did I do something wrong？
These run on  WSA.

**Comment by Howard20181** [comment, marked as answer]
你为什么装远古版本的管理器？
装`/data/adb/lspd`下面的管理器

**Reply by Sunrongguo2008** [reply]
哪里的/data/adb/lspd？

**Comment by Howard20181** [comment]
Android里的

Best regards,
Howard

发件人: ***@***.***>
发送时间: 2022年11月11日 10:33
收件人: ***@***.***>
抄送: Howard ***@***.***>; ***@***.***>
主题: Re: [LSPosed/LSPosed] Why does my LSPosed show that it is not installed? (Discussion #2214)


哪里的/data/adb/lspd？

―
